website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,102 @@
+
+!!!
+
+
+!!! Do
+
+Instead, we recommended replacing the disabled element with contextual guidance to help users understand why this feature isn't available, and providing them with clear steps to enable it.
+
+![Showing an interface with a compact alert explaining why the expected behavior isn't there with guidance.](/assets/patterns/disabled-patterns/conditional-availability-example-explanation.png)
+
+If possible, provide CTAs directly to the place where users can take immediate action.
+
+!!!
+
+
+## Upgrading
+
+When upgradable features are available and a user has the necessary permissions, highlighting a path to upgrade provides a graceful marketing opportunity. However, be mindful to not overdue it and make the user feel overly advertised to.
+
+There are two ways to showcase an upgradable feature:
+
+- Space to explain
+- Interrupting an action
+
+### Space to explain
+
+In this example, "Terraform Health" is a an upgradable feature and is contextually relevant in the workspace. The UI has space to entice the user by explaining a bit of what this service can do for them.
+
+![Screen shot of Terraform workspace highlighting Terraform Health on the side bar below information.](/assets/patterns/disabled-patterns/upgrade-inline-example.png)
+
+### Interrupting an action
+
+In this example, a user is about to perform an action and notices something new within the list of options, one with a "Beta" badge. 
+
+![Showing a dropdown opened, with two list items. One for creating a static secret, while another using an auto-rotating secret with a beta badge next to it.](/assets/patterns/disabled-patterns/upgrade-modal-example-1.png)
+
+Enticed, they click it to use this feature, however, they are not enrolled in this tier and are interrupted by a modal explaining this feature a little more.
+
+![A modal explaining beta to the user, with a CTA to learn more, or close the modal.](/assets/patterns/disabled-patterns/upgrade-modal-example-2.png)
+
+Interrupting the action with a modal helps a user understand more without fully committing, and if more details are necessary they can be directed to an informational page.
+
+
+## Service outage
+
+In instances where outages may occur, provide a clear message to users on what they can do and add a link to [our status page](https://status.hashicorp.com/).
+
+![Displaying Hashicorp's 500 internal error, providing a retry button and a link to the the status page.](/assets/patterns/disabled-patterns/service-outage-example.png)
+
+## Incomplete flow
+
+In a form, always enable the submit button. When buttons are disabled, users are often left confused and frustrated because they can't move forward. Instead, show users what errors occurred so they can fix them and proceed. 
+
+!!! Do
+
+When a user clicks "Create cluster" but has failed to enter a value into a `required` input, an error will display, informing the user how to fix the issue.
```

Comment:
Do we want to use more actionable language for "Do's"?

IE: instead of just providing an example situation, tell them expressly what to do.

"If a user clicks a form submit button but has failed to enter a value in a required field, display an error informing them how to fix the issue."

Reply:
Makes sense, ill update.

Reply:
Updated to what you specified. Resolving.